akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -208,13 +209,24 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
     allocationStrategy: ShardAllocationStrategy,
     handOffStopMessage: Any): ActorRef = {
 
-    requireClusterRole(settings.role)
-    implicit val timeout = system.settings.CreationTimeout
-    val startMsg = Start(typeName, entityProps, settings,
-      extractEntityId, extractShardId, allocationStrategy, handOffStopMessage)
-    val Started(shardRegion) = Await.result(guardian ? startMsg, timeout.duration)
-    regions.put(typeName, shardRegion)
-    shardRegion
+    if (settings.shouldHostShard(cluster)) {
+
+      implicit val timeout = system.settings.CreationTimeout
+      val startMsg = Start(typeName, entityProps, settings,
+        extractEntityId, extractShardId, allocationStrategy, handOffStopMessage)
+      val Started(shardRegion) = Await.result(guardian ? startMsg, timeout.duration)
+      regions.put(typeName, shardRegion)
+      shardRegion
+    } else {
+      system.log.info("Starting Shard Region Proxy [{}] (no actors will be hosted on this node)...", typeName)
```

Comment:
define a more specific logger:
`val log  = Logging(system, this.getClass)`

and use log.debug here